apps/public/src/app/bijlagen/_components/file-upload.module.css

```diff
@@ -0,0 +1,39 @@
+.file-upload-wrapper {
+  position: relative;
+
+  @media screen and (min-width: 36rem) {
+    padding: var(--ams-space-md);
+    border: var(--ams-border-width-sm) dashed var(--ams-color-neutral-grey3);
+    cursor: var(--ams-action-activate-cursor);
```

Comment:
We can move this out of the media query, no? The entire container should always have `cursor: pointer`